.circleci/config.yml

```diff
@@ -22,16 +22,16 @@ jobs:
     working_directory: ~/project
     machine:
       enabled: true
-      # TODO: why is this using Ubuntu instead of Debian?
-      image: ubuntu-2004:202010-01
+      # Because we are running Docker inside this job, we need a machine type.
+      # CircleCI does not currently offer Debian machine images.
+      image: ubuntu-2004:2023.07.1
```

Comment:
Latest per https://circleci.com/developer/images?imageType=machine

Reply:
I don't think this is correct, because CircleCI now offers `setup_remote_docker`, e.g. https://github.com/freedomofpress/securedrop-docs/blob/main/.circleci/config.yml#L37 so we don't strictly need a machine type.

My TODO was more that we're running the tests on Ubuntu rather than Debian,, we could've done that inside a Debian container on top of the Ubuntu VM or something. But I think when we move this over to GHA we can use podman inside the container or use a docker-in-docker setup so that it still runs on Debian.

Reply:
Makes sense. I'm in favor of making that change as part of moving to GitHub Actions, so clarified comment accordingly.